app/src/main/res/values/strings.xml

```diff
@@ -424,7 +423,6 @@
 
   <!--Update Tab-->
   <string name="updatetab_title_installed">Installed</string>
-  <string name="updatetab_button_update_all">Update all</string>
   <string name="updatetab_button_update">Update</string>
   <string name="no_updates_available_retoric">Everything is up to date!</string>
```

Comment:
This string should also be removed since it is only being used on updates fragment